query.go

```diff
@@ -77,11 +80,12 @@ type queryChannelResponseData struct {
 func (c *Client) QueryChannels(q *QueryOption, sort ...*SortOption) ([]*Channel, error) {
 	qp := queryChannelRequest{
 		State:            true,
-		FilterConditions: q,
+		FilterConditions: q.Filter,
 		Sort:             sort,
+		UserID:           q.UserID,
 	}
 
-	data, err := easyjson.Marshal(&qp)
+	data, err := json.Marshal(&qp)
```

Comment:
why is there a change here ?

Reply:
we don't need to use easyjson at all here.
the previous code was broken

Reply:
yes, not needed but sometimes std sometime easyjson can be error prone. 
Since we can't drop it because we need it in some cases, it might be better to stick to it.

What was broken actually?

Reply:
see https://github.com/GetStream/stream-chat-go/pull/60#discussion_r395514655

Reply:
Broken part is actually not marshaling but the type. When above change (`q -> q.filter`), is this still needed?

> we can change to EasyJSON (no strong opinions on this)

I think sticking to one would be better.

Reply:
EasyJSON makes sense for internal stuff where we need to optimism performance. For external stuff, api clients, and the like I'd put my vote for avoiding non-standard dependencies like this.